### ISSUE TITLE
Commas in names break the CSV export

### ISSUE THREAD

**Issue by tjorbo** [Dataport/terminfinder-frontend#69]
**Describe the bug**
Verwendet einE Umfrage-TeilnehmerIn ein Komma im Namen, werden die Daten in dem CSV-Export nicht korrekt dargestellt. Das Komma wird als Ende der Zelle interpretiert, obwohl es übersprungen werden muss.

If a respondent uses a comma in their name, the data will not be displayed correctly in the CSV export. The comma is interpreted as the end of the cell, although it must be skipped.

**To Reproduce**

1. Create a poll
2. Answer the poll with the user name "te,st"
3. Download the csv and look for a broken dataset

**Expected behavior**
Commas should be escaped, so the user can use a comma at will. The csv should not break

**Desktop (please complete the following information):**
- all browsers

**Smartphone (please complete the following information):**
- all browsers

**Additional context**
All exported values like title, description and so on are affected.

**Comment by jase88** [comment]
This ticket is labelled as _good first issue_, but [CONTRIBUTING.md](https://github.com/Dataport/terminfinder-frontend/blob/main/docs/CONTRIBUTING.md) says

> Currently, it is not possible for us to accept merge requests.

How is that to be understood?

**Comment by tjorbo** [comment]
Hi @jase88,
you are absolutely right. I have created a ticket to change the CONTRIBUTING.md and will fix it asap. #182 

In short, contributions are now welcome.

When we created the CONTRIBUTING.md, we didn't have a contributor licence agreement yet, so no contribution could be accepted.

Thanks for addressing this error!

If you wish to contribute on this issue, you are warmly welcome and I can assign it to you.
